front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -67,7 +67,7 @@ const LinearScaleControl = ({
           })}
         </>
         <Box mt="28px" alignSelf="flex-end">
-          <Text mr="8px" fontWeight="bold">
+          <Text mr="8px" color="textSecondary">
             {uischema.options?.maximum_label}
```

Comment:
@benfraserdesign Just wanted to double check how this should look when we have 7 radio buttons or really long label text (or labels of different lengths)? Is this ok?

![image](https://user-images.githubusercontent.com/33987955/196208981-42fbd219-0941-471e-a8f3-868b664082be.png)

Reply:
Didn't expect that TBH, will tweak and make more resilient.